README.md

```diff
@@ -111,8 +111,8 @@ As of now, these are the supported resources:
        --productQueryParameters         Pass your customized product fetch limit
```

Comment:
Won't you think that changing statements like product predicate to product projection predicate would be better, also do we need to document that is querying based on the staged projection?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/268/commits/d9d1a54230e238db5939c59976dd4dcfcdb5e65b
And for documentation regarding query, I don't think here we need to specify it. We are using the same query everywhere to fetch products.